assets/model_monitoring/components/src/data_quality_statistics/run.py

```diff
@@ -22,9 +22,7 @@ def run():
 
     metric_unique_df = compute_data_quality_statistics(df)
 
-    sp_metric_unique_df = (
-        metric_unique_df.to_spark()
-    )  # Convert back to Spark dataframe to output as MLTable
+    sp_metric_unique_df = metric_unique_df
```

Comment:
no need to have this equal? Just rename?

Reply:
removed